web-services/query/src/main/java/datawave/webservice/query/logic/filtered/QueryLogicFilterByAuth.java

```diff
@@ -0,0 +1,101 @@
+package datawave.webservice.query.logic.filtered;
+
+import datawave.webservice.query.Query;
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.accumulo.core.security.ColumnVisibility;
+import org.apache.accumulo.core.security.VisibilityEvaluator;
+import org.apache.accumulo.core.security.VisibilityParseException;
+
+import java.util.Set;
+
+public class QueryLogicFilterByAuth implements FilteredQueryLogic.QueryLogicFilter {
```

Comment:
Please consider adding some javadocs here that describe the intent of the class.